src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicDiscount = 50;
+  const extraDiscount = 20;
+  const rentPrice = 40;
+
+  if (days <= 2) {
+    return days * rentPrice;
+  }
+
+  if (days >= 3 && days <= 6) {
+    return (days * rentPrice) - extraDiscount;
+  }
+
+  if (days <= 7) {
+    return (days * rentPrice) - basicDiscount;
```

Comment:
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item, I don't know what 2, 7 and 3 are